test/files/base64.iced

```diff
@@ -19,6 +19,6 @@ exports.test_b64_compat = (T,cb) ->
     for padlen in [0...pad.length]
       buf = Buffer.concat [ pad[0...padlen], x[0...i] ]
       e1 = b64stripped.encode buf
-      e2 = b64enc.encode buf
+      e2 = b64enc.encode buf, true
```

Comment:
I agree, but then we have to change all the callers - when iced translates into JS, it assigns local variables to the expected pieces of the JSON object, and if the JSON object isn't supplied, it `NameError`s. I don't think it's worth working around given that this will a) be wrapped in a CLI that will handle flags for the callers and b) rarely, if ever be used